### PR TITLE
Add Support for GetAppState Command

### PR DESCRIPTION
**Pull request by broetchenrackete36**
## Change list

Added GetAppState.
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Integration tests
- [ ] Have you provided integration tests to pass against the latest version of appium? (for Bugfix or New feature)

## Details
Added GetAppState-Command (http://appium.io/docs/en/commands/device/app/app-state/).

### CODE REVIEW

File: src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs [hunk 1]
```diff
@@ -60,5 +60,12 @@ public interface IInteractsWithApps : IExecuteMethod
         /// Closes the current app.
         /// </summary>
         void CloseApp();
+
+        /// <summary>
+        /// Gets the State of the app.
+        /// </summary>
+        /// <param name="appId">a string containing the id of the app.</param>
+        /// <returns>an integer number representing the status. 0 is not installed. 1 is not running. 2 is running in background or suspended. 3 is running in background. 4 is running in foreground.</returns>
+        int GetAppState(string appId);
```
Comment (mykola-mokhnach) [inline src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs]: would it be better to have an enumeration for the method result rather than simple int?

Comment (broetchenrackete36) [reply src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs]:
```csharp
public enum AppState
{
    NotInstalled,
    NotRunning,
    RunningInBackgroundOrSuspended,
    RunningInBackgroud,
    RunningInForeground
}
```
Something like this?

File: src/Appium.Net/Appium/AppiumDriver.cs [hunk 2]
```diff
@@ -269,6 +269,9 @@ public void Rotate(Dictionary<string, int> opts)
             Execute(AppiumDriverCommand.BackgroundApp,
                 AppiumCommandExecutionHelper.PrepareArgument("seconds", AppiumCommandExecutionHelper.PrepareArgument("timeout", seconds)));
 
+        public AppState GetAppState(string appId) =>
+            (AppState)Convert.ToInt32(Execute(AppiumDriverCommand.GetAppState, AppiumCommandExecutionHelper.PrepareArgument("appId", appId)).Value.ToString());
```
Comment (KazuCocoa) [inline src/Appium.Net/Appium/AppiumDriver.cs]: [style]
Need a space like `(AppState) Convert`?

**Comment by mykola-mokhnach** [comment]
@akinsolb Do you have time to merge and publish these changes?

**Pull request merged**